doc/python/shapes.md

```diff
@@ -638,6 +638,34 @@ fig.update_layout(
 fig.show()
 ```
 
+#### Adding Shapes to Subplots
+
+```python
+import plotly.graph_objects as go
+from plotly.subplots import make_subplots
+
+# Create Subplots
+fig = make_subplots(rows=2, cols=2, start_cell='bottom-left')
+
+fig.add_trace(go.Scatter(x=[2, 6], y=[1,1]), row=1, col=1)
+fig.add_trace(go.Bar(x=[1,2,3], y=[4,5,6]), row=1, col=2)
+fig.add_trace(go.Scatter(x=[10,20], y=[40,50]), row=2, col=1)
+fig.add_trace(go.Bar(x=[11,13,15], y=[8,11,20]), row=2, col=2)
+
+# Add shapes
+fig.update_layout(
+    shapes=[
+        dict(type="line", xref="x1", yref="y1",
+            x0=3, y0=0.5, x1=5, y1=0.8),
```

Comment:
```suggestion
            x0=3, y0=0.5, x1=5, y1=0.8, line_width=3),
```

Reply:
the line width was too small, I did not see the line and did not understand why this one was not there :-)